src/components/link-detail/index.js

```diff
@@ -275,6 +275,11 @@ export class LinkDetail extends React.Component {
                 placeholder="username"
                 value={this.state.upstreamOwner}
                 onChange={e => {
+                  // Remove the `github.com`, if the user pasted a full GitHub link
+                  var githubMatchExpression = new RegExp("https?:\\/\\/github\\.com\\/", "")
```

Comment:
Can you use `const` instead of `var`, also use a literal regex instead of the constructor?

For example, `const githubMatchExpression = /https?:\/\/github\.com\//;`

Even better, given that the identical regular expression is used in both locations, it might be better for this to live in the root of the module (ie, not in the component) to be more DRY.

Reply:
I tried to remove the duplicated code, perhaps it is correct.